server/controllers/profile.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Controller.
+ */
+module.exports = function(app) {
+
+  const models = app.set('models');
+  const errors = app.errors;
+
+  return (req, res, next) => {
+    const slug = req.params.slug;
+    const controllers = req.app.set('controllers');
+
+    models.Group
+      .findOne({where: { slug }})
+      .then((group) => {
+        if(group) {
+          req.group = group;
+          return controllers.groups.getOne(req, res, next);
+        }
+        else {
+          models.User.findOne({where: {username: slug}})
+          .then((user) => {
+            if(user) {
+            req.user = user;
+            req.query.profile = true;
+            return controllers.users.show(req, res, next);
```

Comment:
I agree and that's e.g how GitHub does, e.g github.com/sedubois vs github.com/opencollective. But we then need to ensure unicity across group slugs + usernames.